src/apis/Global.re

```diff
@@ -1,7 +1 @@
-[@bs.val] external __DEV__: bool = "__DEV__";
 
```

Comment:
This file should not untracked not removed

Reply:
@jfrolich Could you revise your PR to keep this file. I believe it's ready to be merged otherwise.